specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -69,53 +69,53 @@ paths:
           schema:
             type: "string"
         - in: query
-          name: appartementsrechtvolgnummer
+          name: kadastraleaanduiding__appartementsrechtvolgnummer
           description: "Nummer dat het kadastraal object uniek identificeert als een\
             \ appartementsrecht binnen het complex."
           schema:
             type: "integer"
         - in: query
-          name: burgerservicenummer
+          name: ingeschrevenpersoon__burgerservicenummer
           description: "Het burgerservicenummer van de persoon die een zakelijk recht op een kadastraal onroerende zaak heeft. Deze persoon is zakelijk gerechtigde van de kadastraal onroerende zaak. Door deze query-parameter te gebruiken worden Kadastraal Onroerende Zaken geretourneerd waar deze persoon een zakelijk recht op heeft."
           required: false
           schema:
             type: string
             maxLength: 9
         - in: query
-          name: kadasterpersoonidentificatie
+          name: persoon_kadasterpersoonidentificatie
           description: "De identificatie van de Kadasterpersoon, een niet-ingeschreven persoon of een rechtspersoon waarvan de gegevens door het kadaster zijn geregistreerd. Deze persoon is zakelijk gerechtigde van de Kadastraal Onroerende Zaak. Door deze query-parameter te gebruiken worden Kadastraal Onroerende Zaken geretourneerd waar deze persoon een zakelijk recht op heeft."
           required: false
           schema:
             type: string
         - in: query
-          name: typegerechtigde
+          name: zakelijkgerechtigde__typegerechtigde
           description: "Een typering van het recht dat de zakelijkgerechtigde heeft op de Kadastraal Onroerende Zaak. Door het gebruik van deze query-parameter worden Kadastraal Onroerende Zaken geretourneerd waar een recht op rust van het opgegeven type."
           required: false
           schema:
             $ref: "#/components/schemas/typeGerechtigde_enum"
         - in: query
-          name: postcode
+          name: adres__postcode
           description: "De postcode van het adres van de objectlocatie van de kadastraal onroerende zaak"
           required: false
           schema:
             type: string
             pattern: "^[1-9][0-9][0-9][0-9][A-Z][A-Z]$"
         - in: query
-          name: huisnummer
+          name: adres__huisnummer
```

Comment:
adres is geen eigenschap van de kadastraalonroerendezaak. We hebben "nummeraanduidingIdentificatie" en we hebben en link naar "bagAdressen".

Neemt niet weg dat "adres" n.m.m. eigenlijk beter (duidelijker en bondiger) is. Maar dan zou de link ook "adressen" moeten heten i.p.v. "bagAdressen".

Maar aangezien het toch niet meer exact verwijst naar een property in de resource, wat is dan nog de toegevoegde waarde van de prefix? Ik denk dat iedereen wel snapt dat een postcode of huisnummer onderdeel is van een adres.

Reply:
> Maar aangezien het toch niet meer exact verwijst naar een property in de resource, wat is dan nog de toegevoegde waarde van de prefix? Ik denk dat iedereen wel snapt dat een postcode of huisnummer onderdeel is van een adres.

Dit is ook een manier om met de prefix om te gaan. Alleen kan je dan geen onderscheid maken tussen parameters die in de resource zitten en parameters die nergens in de resource, of gerelateerde resource zijn opgenomen. Misschien dan een standaard prefix gebruiken om duidelijk te maken dat de parameter geen onderdeel is van de opgevraagde resource ?in feite nergens in je schema voorkomen.